setup.py

```diff
@@ -157,37 +157,39 @@ def fetch_path_from_blender_script(blender_exec, script, keyword="blender"):
-    out = Path(exec_shell_cmd([cmd, "blender"]))
-    if out.exists() and "blender" in str(out).lower():
-        if input(f"Take this blender version '{out}'? (y/n)>").lower() in ["y", "yes"]:
-            auto_found = True
-        break
-while not auto_found or not out.exists() or not "blender" in str(out).lower():
-    auto_found = True
-    print("Could not retrieve blender executable from this path:", out)
-    out = Path(input("Please provide the path to your blender executable: ").strip())
-blender_path = deepcopy(out)
-scripts_path = fetch_path_from_blender_script(blender_path, this_dir/"get_blender_path.py")
-modules_path = scripts_path / "modules"
-addons_path = scripts_path / "addons"
-print("Found the following blender installation directories:")
-print(scripts_path)
-print(modules_path)
-print(addons_path)
-if check_blender_agreement(input("Do you want to proceed? (y/n)>").strip()):
-    #install phobos
-    target = addons_path/"phobos"
-    if target.exists():
-        shutil.rmtree(target)
-    shutil.copytree(this_dir/"phobos", target)
-    # install requirements
-    python_path = fetch_path_from_blender_script(blender_path, this_dir/"get_blender_python.py")
-    check_requirements(python_path, upgrade_pip=True, lib=modules_path)
+
+if False:
+    if "AUTOPROJ_CURRENT_ROOT" in os.environ:
+        sys.exit()
+
+    blender_path = None
+    print("Thanks for installing phobos!\n\nTrying now to install blender addon")
+    check_blender_agreement(input("Do you want to proceed? (y/n)>").strip())
+    auto_found = False
+    for cmd in ["which", "where", "Get-Command"]:
+        out = Path(exec_shell_cmd([cmd, "blender"]))
+        if out.exists() and "blender" in str(out).lower():
+            if input(f"Take this blender version '{out}'? (y/n)>").lower() in ["y", "yes"]:
+                auto_found = True
+            break
+    while not auto_found or not out.exists() or not "blender" in str(out).lower():
+        auto_found = True
+        print("Could not retrieve blender executable from this path:", out)
+        out = Path(input("Please provide the path to your blender executable: ").strip())
+    blender_path = deepcopy(out)
+    scripts_path = fetch_path_from_blender_script(blender_path, this_dir/"get_blender_path.py")
+    modules_path = scripts_path / "modules"
+    addons_path = scripts_path / "addons"
+    print("Found the following blender installation directories:")
+    print(scripts_path)
+    print(modules_path)
+    print(addons_path)
+    if check_blender_agreement(input("Do you want to proceed? (y/n)>").strip()):
+        #install phobos
+        target = addons_path/"phobos"
+        if target.exists():
+            shutil.rmtree(target)
+        shutil.copytree(this_dir/"phobos", target)
+        # install requirements
+        python_path = fetch_path_from_blender_script(blender_path, this_dir/"get_blender_python.py")
+        check_requirements(python_path, upgrade_pip=True, lib=modules_path)
```

Comment:
Please do not commit changes you have only used for your local version, this is not a valid change

Reply:
I have resolved this via merging dfki-ric/phobos:merge_cli

Reply:
Thanks